modules/options/src/main/scala/scala/build/options/ScalacOpt.scala

```diff
@@ -18,7 +18,8 @@ object ScalacOpt {
   private val repeatingKeys = Set(
     "-Xplugin",
     "-P", // plugin options
-    "-language"
+    "-language",
+    "-Wconf"
```

Comment:
This one again :/ Feels like the logic is wrong here, the default behaviour should be to repeat things unless it's one of the options that can't be repeated.

Reply:
I mean, the proper fix is to work on https://github.com/VirtusLab/scala-cli/issues/2867 first.

Reply:
The problem is that we're emulating `scalac` options as `scala-cli` options, and hardcoding special cases for now.
We can feed it from the compiler settings API, but that requires rewriting a considerable chunk of the logic.

Reply:
It seems I failed to include every repeating option, but another issue I see is that this was meant to allow overwriting directives with cli options, but not necessarily cli options with another cli options (or directive with another directive). Apologies for the error!

Reply:
No worries, it's also hard to keep track of it from one compiler version to another. 
We need to fix it properly in #2867.